Corona-Warn-App/src/main/res/values-de/antigen_strings.xml

```diff
@@ -170,4 +170,15 @@
     <string name="rat_profile_create_phone_hint">Telefonnummer</string>
     <!-- XTXT: Create RAT profile email hint -->
     <string name="rat_profile_create_email_hint">E-Mail-Adresse</string>
+
+    <!--RAT profile onboarding-->
+    <string name="rat_profile_onboarding_image_content_description">One person wearing a mask, standing in front of a Corona test lab and showing her profile Qr Code on phone</string>
```

Comment:
@janetback  Please let me know that is the right text for DE and EN for the illustration accessibility description

Reply:
"Eine Frau mit einem Smartphone in der Hand steht vor einem Gebäude. Ein QR-Code symbolisiert den zu scannenden Schnelltest-Profil."